packages/mdl-fab/mdl-fab.scss

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "variables";
+
+.mdl-fab {
```

Comment:
A few points on elevation:

* The FAB has a resting elevation of 6dp, and a pressed elevation of 12dp (with a transition in between with the same properties as regular buttons). See the chart at https://material.google.com/what-is-material/elevation-shadows.html#elevation-shadows-elevation-android-
* The FABs shadow shouldn't change when it's focused.